wp-content/themes/workingnyc/search.php

```diff
@@ -14,17 +14,14 @@
 // Create query
 $wp_query = new WP_Query(array(
   's' => $term,
-  'post_type' => 'any'
+  'post_type' => array('programs', 'jobs')
 ));
 
 // Redo relevanssi query and get posts in Timber format.
 // This block could be made more efficient.
 $relevanssi_query = relevanssi_do_query($wp_query);
```

Comment:
We use the [Relevanssi](https://www.relevanssi.com/user-manual/functions/relevanssi_do_query/) plugin here to get better search results